lib/data/rates/energy_grants_calculator.yml

```diff
@@ -0,0 +1,3 @@
+- start_date: 2015-04-06
+  end_date: 3000-01-01
```

Comment:
We don't actually know when the rates change. The threshold dates don't follow a pattern:
* 2012-2013: 5th July, 1951
* 2013-2014: 5th January, 1952
* 2014-2015: 5th January, 1953
* 2015-2016: 5th May, 1953

I can change the start date to the beginning of the current tax year (2016-04-06), but as I can't guarantee the rates will be updated by the end of the tax year I don't want to run the risk of users not being aware that they "could" make a claim for payment, so set the end date to the "end of time".